test/controllers/overseas.name.controller.spec.ts

```diff
@@ -270,7 +270,7 @@ describe("Overseas Name controller", () => {
 
     test(`redirect to the ${PRESENTER_PAGE} page with transaction and overseas entity already created`, async () => {
       mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_SAVE_AND_RESUME
-      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_ROE_UPDATE
+      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_REDIS_REMOVAL
```

Comment:
Another mock flag has been added here, is this now testing remove?

Reply:
> Another mock flag has been added here, is this now testing remove?

The flags have been renamed because when Martin and I checked to see which flag was being checked/called for, turned out to be FEATURE_FLAG_ENABLE_REDIS_REMOVAL.